src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -1318,7 +1318,8 @@ async function webviewPreloads(ctx: PreloadContext) {
 			}
 
 			this._api = await module.activate(this.createRendererContext());
-
+			// Preloads need to be loaded before loading renderers.
+			await kernelPreloads.waitForAllCurrent();
```

Comment:
I think this should happen at the start of this function (before the import and activate)